templates/definition/charger/tapo.yaml

```diff
@@ -2,21 +2,31 @@ template: tapo
 products:
   - brand: TP-Link
     description:
-      generic: Tapo
+      generic: Tapo P-Series Smart Plug
 group: switchsockets
 params:
   - name: uri
     required: true
+    example: http://192.0.2.2
+    help:
+      en: tapo device ip address (local)
+      de: Die IP-Adresse der Tapo Steckdose im lokalen LAN/WLAN
   - name: user
     required: true
+    help:
+      en: user of TP-Link account (usually an e-mail address)
+      de: TP-Link Benutzerkonto (Normalerweise die E-Mail Adresse, wie in der Tapo App konfiguriert)
   - name: password
     required: true
     mask: true
+    help:
+      en: password of of TP-Link account
+      de: Password des TP-Link Benutzerkontos
   - name: standbypower
     default: 15
 render: |
   type: tapo
-  uri: {{ .uri }}
-  user: {{ .user }}
-  password: {{ .password }}
+  uri: http://{{ .host }}  # tapo device ip address (local)
```

Comment:
Hier wird `host` verwendet, oben aber `uri` vom Nutzer abgefragt.

Die Kommentare in Render können raus (Redundanz).